src/DesktopHeader.jsx

```diff
@@ -88,7 +89,7 @@ class DesktopHeader extends React.Component {
           className="btn btn-outline-primary d-inline-flex align-items-center pl-2 pr-3"
         >
           <Avatar size="1.5em" src={avatar} alt="" className="mr-2" />
-          {username} <CaretIcon role="img" aria-hidden focusable="false" />
+          {name || username} <CaretIcon role="img" aria-hidden focusable="false" />
```

Comment:
This would make so there's no longer a 1:1 relationship between the DesktopHeader props and what is actually shown.  In other words, it's not immediately obvious without looking at the implementation what happens if you provide both username and name.

In other words, unless I'm missing something, DesktopHeader has no reason to distinguish between `username` and `name`.   It should just use a single prop, and the business logic of what to actually put there should be up to the user of the component.

Reply:
@arbrandes That makes sense. Updated!

Reply:
@arbrandes Gentle reminder.